src/View/SSViewer_Scope.php

```diff
@@ -339,6 +344,9 @@ public function next()
     public function __call($name, $arguments)
     {
         $on = $this->itemIterator ? $this->itemIterator->current() : $this->item;
+        if (is_scalar($on) || is_null($on)) {
+            $on = $this->convertPrimitiveToDBField($on);
+        }
```

Comment:
```suggestion
        if (is_scalar($on)) {
            $on = $this->convertPrimitiveToDBField($on);
        }
```
`null` is already correctly handled - it's an empty value and shouldn't be rendered as though it were a string.

Reply:
Done